DOC Audit security config change (#66839)
elastic/elasticsearch: x-pack/docs/en/security/auditing/auditing-search-queries.asciidoc

request body can contain malicious content that can break a parser consuming
the audit logs.

The request body is printed as an escaped JSON string value (RFC 4627) to the `request.body`
event attribute.

Not all events contain the `request.body` attribute, even when the above setting
is toggled. The ones that do are: `authentication_success`,
`authentication_failed`, `realm_authentication_failed`, `tampered_request`, `run_as_denied`,

> and `anonymous_access_denied` that are printed on the coordinating node only

Reviewer: Minor suggestion, but I think this sentence can be split:
```suggestion
and `anonymous_access_denied`. The `request.body` attribute is printed on the coordinating node only
```

> (the node that handles the REST request). Most of them are

Reviewer 2: ```suggestion
(the node that handles the REST request). Most of these event types are
```
